src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Cognitive Services management functions for managing Microsoft Azure Cognitive Services accounts.")]
 
-[assembly: AssemblyVersion("0.2.0.0")]
-[assembly: AssemblyFileVersion("0.2.0.0")]
+[assembly: AssemblyVersion("0.2.1.0")]
```

Comment:
We have publish version 0.2.0.0 in NuGet already, if I keep it as 0.2.0.0, will we be able to publish it again?